packager/packager.cc

```diff
@@ -686,8 +686,9 @@ vector<ast::ParsedFile> Packager::run(core::GlobalState &gs, WorkerPool &workers
                         if (auto pkg = constPkgDB.getPackageForContext(ctx)) {
                             job = rewritePackagedFile(ctx, move(job), pkg->name.mangledName);
                         } else {
-                            // Don't transform, but raise an error.
-                            if (auto e = ctx.beginError(job.tree->loc, core::errors::Packager::UnpackagedFile)) {
+                            // Don't transform, but raise an error on the first line.
```

Comment:
Our testsuite doesn't support multiline errors, so this is the only behavioral change in this PR.